generators/app/templates/src/config/pushNotifications.js

```diff
@@ -1,6 +1,6 @@
-let PushNotification;
+let pushNotifications;
 try {
-  PushNotification = require('react-native-push-notification'); // eslint-disable-line global-require
+  pushNotifications = require('react-native-push-notification'); // eslint-disable-line global-require
```

Comment:
⚠️ Wrong changes in this file, revert them! You just had to rename the file. Also, you missed change the name of `reactotronConfig.js` and `huaweiPushNotifications.js` too.

Reply:
😳